MODULE.bazel

```diff
@@ -0,0 +1,15 @@
+"Define the Bazel module published to the Bazel Central Registry"
+
+module(
+    name = "rules_ruby",
+    compatibility_level = 1,
+    # NB: this version will be replaced after a release by the Publish to BCR app
+    version = "0.0.0",
+)
+
+# Lower-bound dependency versions.
+# These should NOT be increased unless needed, as bumping our lower-bound may change
+# the versions resolved in users repositories.
+bazel_dep(name = "platforms", version = "0.0.5")
+
+# TODO: should we register any toolchain by default?
```

Comment:
I would say we should not unless the user explicitly needs it, what do you think?

Reply:
The user will be forced to register *something* as the toolchain, or they just get build errors when Bazel fails to resolve the toolchain type to anything.

Probably this TODO should stay here for this initial PR, but I'd like to discuss in the future, as it's more user friendly if you just need to `bazel_dep(name = "rules_ruby")` and now you can already run a basic ruby program

Reply:
As a casual user, I'd probably want the JRuby toolchain to get registered by default so I don't have to bother with waiting on compiling a Matz Ruby from sources. Only when I run into a gem with native dependencies or a program that depends on MRI specific behavior would I really need to dig into documentation and figure out how to override with a different toolchain.

Reply:
> As a casual user, I'd probably want the JRuby toolchain to get registered by default so I don't have to bother with waiting on compiling a Matz Ruby from sources.

I suppose it could be a compromise until there is a portable version of MRI.

> Only when I run into a gem with native dependencies or a program that depends on MRI specific behavior would I really need to dig into documentation and figure out how to override with a different toolchain.

This might be tricky to figure out, but in any event, we can start with JRuby to avoid any slow toolchain installation.